packages/paste-website/src/pages/components/card/index.mdx

```diff
@@ -56,71 +87,166 @@ export const pageQuery = graphql`
 
 ### About Cards
 
-Cards are a specifically-styled container that group relatable content and actions.
-Card is an extremely flexible container that does not require specific components inside of it,
-allowing an implementer to compose as needed, but elements such as Title, Body,
-and Button or Anchor are commonly used. See Examples for common Card patterns.
-A Card does not currently handle any interactive events such as hover, click or focus,
-though children composed inside of it may commonly have event handlers.
+Cards are specifically-styled containers that group relatable content and actions. Card is an extremely 
+flexible container that does not require specific components inside of it, allowing an implementer to 
+compose as needed, but elements such as Heading, Text, and Button or Anchor are commonly used.
+See [When to use a Card](#when-to-use-a-card) and [Examples](#examples) for common Card patterns and dos/dont’s. A Card does not 
+currently handle any interactive events such as hover, click, or focus, though children 
+composed inside of it may commonly have event handlers.
 
-Acknowledging the flexibility of Card component, there are several non-negotiable
-properties of a Card that differentiate it from a more basic page-layout element, such as Box:
+Acknowledging the flexibility of Card component, there are several non-negotiable properties 
+of a Card that differentiate it from a more basic page-layout element, such as [Box](/utilities/box):
 
 * A background color of $color-background-body.
 * A border width of $border-width-20.
 * A border radius of $border-radius-20.
-* A border color of $color-border.
+* A border color of $color-border-light.
 * A flexible width and height.
     * Height is determined by a Card’s child components
     * Width is determined by the parent component or container surrounding Card
 
 ### Card vs. Box
 
-A card is a box with default attributes defined above. The purpose of the card is to provide a consistent experience to use across the website the design team can
+At its core, Card is a Box with styled with the specific default attributes above and more explicit use cases 
```

Comment:
I think this first sentence could do with some work

Reply:
At its core, Card is a `<Box>` with specific styling attributes and more explicit use cases and that you can find in [Examples](/).